SonarQube.Bootstrapper/IBootstrapperSettings.cs

```diff
@@ -35,6 +37,16 @@ public interface IBootstrapperSettings
         string PostProcessorFilePath { get; }
 
         /// <summary>
+        /// Full path to the xml file containing the supported bootstrapper versions
```

Comment:
".. containing the logical bootstrapper API versions supported by the pre/post-processors"